### PR TITLE
boards: we: added board file orthosie1ev

### PR DESCRIPTION
**Pull request by mah-eiSmart**
Added new board file of Wurth Electronic board Orthosie-I

### CODE REVIEW

File: boards/we/orthosie1ev/we_orthosie1ev.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - watchdog
+  - uart
+  - dma
+  - pwm
+  - spi
+  - counter
+  - entropy
+testing:
+  ignore_tags:
+    - net
```
Comment (kartben) [inline boards/we/orthosie1ev/we_orthosie1ev.yaml:21]: Will need to be removed if/when https://github.com/zephyrproject-rtos/zephyr/pull/80785 gets merged

**Review by kartben — DISMISSED** [review]
Great work!

File: soc/espressif/esp32c3/Kconfig.soc [hunk 2]
```diff
@@ -19,6 +19,12 @@ config SOC_ESP32C3_MINI_N4
 	help
 	  ESP32C3_MINI_N4
 
+config SOC_ESP32C3_ORTHOSIE1
```
Comment (sylvioalves) [inline soc/espressif/esp32c3/Kconfig.soc:22]: Please, remove this entry and use `SOC_ESP32C3` in the board definition.

Comment (mah-eiSmart) [reply soc/espressif/esp32c3/Kconfig.soc:22]: Thank you, yes, that makes sense. I will push the updated version.

**Comment by mah-eiSmart** [comment]
Thank you @sylvioalves and @kartben for reviewing that quick!

**Review by sylvioalves — DISMISSED** [review]

File: boards/we/orthosie1ev/doc/img/we_orthosie1ev.webp
Comment (nordicjm) [inline boards/we/orthosie1ev/doc/img/we_orthosie1ev.webp:1]: put file through https://tinypng.com/

File: boards/we/orthosie1ev/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+
+- Cryptographic hardware acceleration (RNG, ECC, RSA, SHA-2, AES)
+
+For more information, check the datasheet at `ESP32-C3 Datasheet`_ or the technical reference
+manual at `ESP32-C3 Technical Reference Manual`_.
+
+Supported Features
+==================
+
+Current Zephyr's Orthosie-I-EV board supports the following features:
```
Comment (nordicjm) [inline boards/we/orthosie1ev/doc/index.rst:45]: replace board name with the name used when running west build, with \`\` around it and refer to it as board target

File: boards/we/orthosie1ev/we_orthosie1ev.dts [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-names = "default";
+};
+
+&usb_serial {
+	/* requires resoldering of resistors on the board */
```
Comment (nordicjm) [inline boards/we/orthosie1ev/we_orthosie1ev.dts:51]: if this is needed, should it be enabled by default?

**Review by wmrsouza — DISMISSED** [review]

**Comment by rettichschnidi** [comment]
Would change the commit subject to `boards: we: Add board orthosie1ev`

**Comment by mah-eiSmart** [comment]
Thank you @wmrsouza, @rettichschnidi for feedback. The last push-force has your changes applied.

**Review by nordicjm — COMMENTED** [review]
stray `f` in commit message

File: boards/we/orthosie1ev/doc/index.rst [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+
+- Cryptographic hardware acceleration (RNG, ECC, RSA, SHA-2, AES)
+
+For more information, check the datasheet at `ESP32-C3 Datasheet`_ or the technical reference
+manual at `ESP32-C3 Technical Reference Manual`_.
+
+Supported Features
+==================
+
+Current Zephyr's ``we_orthosie1ev`` board supports the following features:
```
Comment (nordicjm) [inline boards/we/orthosie1ev/doc/index.rst:45]:
```suggestion
Currently Zephyr's ``we_orthosie1ev`` board target supports the following features:
```

**Comment by mah-eiSmart** [comment]
Applied changes from @nordicjm. Thank you for the comments.

**Review by nordicjm — APPROVED** [review]

**Review by sylvioalves — APPROVED** [review]

**Review by wmrsouza — APPROVED** [review]

**Comment by mah-eiSmart** [comment]
@kartben , who can merge the PR now? Thx and have a nice day.

**Pull request merged**